app/models/user.rb

```diff
@@ -433,6 +438,14 @@ def is_new?
     Time.current - self.created_at <= NEW_USER_DAYS.days
   end
 
+  def add_or_update_keybase_proof(kb_username, kb_signature)
+    self.keybase_signatures ||= []
+    self.keybase_signatures = self.keybase_signatures.select do |kbsig|
```

Comment:
```suggestion
    self.keybase_signatures.reject! {|kbsig| kbsig['kb_username'] == kb_username }
```

One-liner, also makes it explicit that we're throwing away any existing sig with this username.

Reply:
agreed. this is strictly better.